app/controllers/application_controller.rb

```diff
@@ -1,2 +1,5 @@
 class ApplicationController < ActionController::Base
+  def current_user
+    User.first
+  end
 end
```

Comment:
- [ ] Great job installing your Devise gem! Let's remove the `current_user` method as Devise will provide it.